apps/design/backend/src/language_and_audio/README.md

```diff
@@ -0,0 +1,17 @@
+# Language and Audio
+
+## Google Cloud Authentication
+
+We've created a
+[Google Cloud service account](https://cloud.google.com/iam/docs/service-account-overview)
+for VxDesign that has access to the Google Cloud Translation and Text-to-Speech
+APIs. VxDesign will be deployed with the VxDesign service account key.
+
+For local development, VotingWorks employees can impersonate the VxDesign
+service account. To do this, you'll need to
+[install the Google Cloud CLI](https://cloud.google.com/sdk/docs/install-sdk)
+and run the following command:
+
+```sh
+gcloud auth application-default login --impersonate-service-account vxdesign@astral-pursuit-395520.iam.gserviceaccount.com
+```
```

Comment:
Here's a link to the VxDesign service account in Google Cloud: https://console.cloud.google.com/iam-admin/serviceaccounts/details/115402684050827579647?project=astral-pursuit-395520

I didn't add this link to the README because 1) few should ever have to modify the service account and 2) I get the sense that these Google Cloud URLs may change and aren't designed to be permanently linked to.

Regarding the service account key, I've put that in 1Password.

That said, it seems likely that we'll create a new service account + key under a production Google Cloud project (as opposed to the current test project) when we're actually ready to deploy VxDesign.